src/core/matter/mod.rs

```diff
@@ -483,6 +483,16 @@ pub(crate) trait Matter: Default {
 
         Ok(())
     }
+
+    fn raw_size(&self) -> Result<u32> {
+        let sizes = tables::sizage(&self.code())?;
+        if sizes.fs != 0 {
+            Ok(sizes.fs)
+        } else {
+            let fs = sizes.hs + sizes.ss + (self.size() * 4);
+            Ok(fs)
+        }
+    }
```

Comment:
https://github.com/WebOfTrust/keripy/blob/development/src/keri/core/coring.py#L871-L881

Doesn't seem to match?

Reply:
Good catch! 
Added commit with corrected expression